example/app/controllers/authenticated_controller.rb

```diff
@@ -1,5 +0,0 @@
-class AuthenticatedController < ApplicationController
```

Comment:
Yeah this app is in the Engine so they can access it already. Developers just need to inherit from this controller for the most part anyways. In a real app you might copy it over if you need to add more methods but it doesn't serve a purpose in the example.